Machine.Specifications.ReSharper.Provider.10/MspecTestElementMapper.cs

```diff
@@ -6,13 +6,12 @@
 using JetBrains.ProjectModel;
 using JetBrains.ReSharper.Resources.Shell;
 using JetBrains.ReSharper.UnitTestFramework;
-using JetBrains.ReSharper.UnitTestFramework.DotNetCore;
 using JetBrains.ReSharper.UnitTestFramework.DotNetCore.Common;
 using JetBrains.Util.Extension;
 
 namespace Machine.Specifications.ReSharperProvider
 {
-    public class MspecTestElementMapper : DotNetCoreTestElementMapper
+    public class MspecTestElementMapper : ITestElementMapper
```

Comment:
Inherit from `TestElementMapper`, and the only change you need below is to make `Map` public instead of protected. Rest of the changes in this file are unnecessary.